### PR TITLE
Adds home run baseball burger

### PR DESCRIPTION
**Pull request by PKPenguin321**
Now you can home run it up with a fantastic burger sprited by myself yes
yes
like and subscribe :+1: 

:cl:
add: Chef, do your station a favor and make a home run baseball burger (TM)
/:cl:

### CODE REVIEW

File: code/modules/food_and_drinks/recipes/tablecraft/recipes_burger.dm [hunk 1]
```diff
@@ -253,3 +253,12 @@
 	)
 	result = /obj/item/weapon/reagent_containers/food/snacks/burger/rat
 	category = CAT_BURGER
+
+/datum/crafting_recipe/food/ratburger
```
Comment (optimumtact) [inline code/modules/food_and_drinks/recipes/tablecraft/recipes_burger.dm]: oi

Comment (PKPenguin321) [inline code/modules/food_and_drinks/recipes/tablecraft/recipes_burger.dm]: sssshit i changed this but didnt save,

**Comment by PKPenguin321** [comment]
[Common] Lauser McMauligan says, "This shift was pretty long"
[Common] Lauser McMauligan says, "This shift hasbeen about an hour and five minutes"
[Common] Braids Grenades says, "AN HOUR AND FIVE MINUTES HE SAYS"
[Common] Braids Grenades says, "LIKE I'M GONNA TAKE THE TIME OF DAY FROM A GUY WHO THINKS A BASEBALL BAT ON A BUN IS A MEAL"

File: code/modules/food_and_drinks/food/snacks_burgers.dm [hunk 2]
```diff
@@ -196,4 +196,10 @@
 	name = "rat burger"
 	desc = "Pretty much what you'd expect..."
 	icon_state = "ratburger"
-	bonus_reagents = list("nutriment" = 1, "vitamin" = 1)
\ No newline at end of file
+	bonus_reagents = list("nutriment" = 1, "vitamin" = 1)
+
+/obj/item/weapon/reagent_containers/food/snacks/burger/baseball
+	name = "home run baseball burger"
+	desc = "It's still warm. The steam coming off of it looks like baseball."
```
Comment (ShadowDeath6) [inline code/modules/food_and_drinks/food/snacks_burgers.dm]: Is this desc intended to be worded like this?

Comment (PKPenguin321) [inline code/modules/food_and_drinks/food/snacks_burgers.dm]: Yes.

**Pull request merged**